diff --git a/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js b/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
index e0c9b34..36fb4e3 100644
--- a/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
+++ b/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
@@ -101,7 +101,7 @@ class singleH1Assessment extends _assessment2.default {
 		}
 
 		// Returns the default assessment result if there is one H1 and it's at the beginning of the body.
-		if (this._h1s.length === 1 && this.firstH1AtBeginning()) {
+		if (this._h1s.length === 1) {
 			return;
 		}
 
