diff --git a/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js b/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
index e0c9b34..36fb4e3 100644
--- a/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
+++ b/node_modules/yoastseo/src/assessments/seo/SingleH1Assessment.js
@@ -101,7 +101,7 @@ class singleH1Assessment extends _assessment2.default {
 		}
 
 		// Returns the default assessment result if there is one H1 and it's at the beginning of the body.
-		if (this._h1s.length === 1 && this.firstH1AtBeginning()) {
+		if (this._h1s.length === 1) {
 			return;
 		}
 
diff --git a/node_modules/yoastseo/src/worker/AnalysisWebWorker.js b/node_modules/yoastseo/src/worker/AnalysisWebWorker.js
index e5d905e..c60202a 100644
--- a/node_modules/yoastseo/src/worker/AnalysisWebWorker.js
+++ b/node_modules/yoastseo/src/worker/AnalysisWebWorker.js
@@ -867,7 +867,7 @@ class AnalysisWebWorker {
   */
 	async analyze(id, { paper, relatedKeywords = {} }) {
 		// Automatically add paragraph tags, like Wordpress does, on blocks padded by double newlines or html elements.
-		paper._text = (0, _autop.autop)(paper._text);
+		// paper._text = (0, _autop.autop)(paper._text);
 		paper._text = string.removeHtmlBlocks(paper._text);
 		const paperHasChanges = this._paper === null || !this._paper.equals(paper);
 		const shouldReadabilityUpdate = this.shouldReadabilityUpdate(paper);
